src/cljs/rems/administration/change_catalogue_item_form.cljs

```diff
@@ -77,11 +77,6 @@
                   (all-items-have-the-form-already? items form))}
    (text :t.administration/change)])
 
-(defn- back-button []
-  [atoms/link {:class "btn btn-primary"}
```

Comment:
Btw. "back" button on this page used to be btn-primary, this patch now makes it btn-secondary. Not a huge problem, but makes me think of how the workflow of using this page goes. I think it would be more consistent with the other pages that allow making changes if this back button was "cancel" button instead, and making the change to the form of the catalogue item would automatically link the user back to catalogue items page -- as far as I remember, it currently stays on the page indefinitely until the user either presses "back" button or navigates elsewhere.

In any case, not a problem introduced by this patch, so doesn't need to be changed now.